### PR TITLE
Results cards - Bid List button and Bid Counts

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds the Bid List button and Bid Counts to the results cards.

Relies on #1855 and #1838.

### CODE REVIEW

File: src/Components/ResultsContainer/ResultsContainer.jsx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import PropTypes from 'prop-types';
 import PaginationWrapper from '../PaginationWrapper/PaginationWrapper';
 import ResultsList from '../ResultsList/ResultsList';
 import { POSITION_SEARCH_RESULTS, EMPTY_FUNCTION, SAVED_SEARCH_MESSAGE, SAVED_SEARCH_OBJECT,
```
Comment (Icemanbeta) [inline src/Components/ResultsContainer/ResultsContainer.jsx]: I'm surprised eslint lets you do this, I swear I did this before and it say the tab length was incorrect. I think there's an `/* eslint-disable */ somewhere persisting. No complaints here though lol

File: src/Components/ResultsContainer/ResultsContainer.test.jsx [hunk 2]
```diff
@@ -70,6 +72,8 @@ describe('ResultsContainerComponent', () => {
       newSavedSearchHasErrored={false}
       newSavedSearchIsSaving={false}
       resetSavedSearchAlerts={() => {}}
```
Comment (Icemanbeta) [inline src/Components/ResultsContainer/ResultsContainer.test.jsx]: This goes back to the eslint thing, I swear I've seen eslint errors for not having a default value on required props.

image
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react.js

But actually, why not just set the default value to an empty function. Seems like it doesn't hurt.

File: src/sass/_results.scss [hunk 3]
```diff
@@ -147,7 +177,7 @@ $results-container-width: 100% - $filter-container-width;
       }
     }
 
-    .usa-button {
```
Comment (Icemanbeta) [inline src/sass/_results.scss]: Does this work in IE10?

https://caniuse.com/#search=%3Anot

Is there better browser compatibility documentation somewhere? I feel like I don't trust caniuse. But anyway, according to this it doesn't support selector format but it does if it's an element?

File: src/Components/ResultsContainer/ResultsContainer.test.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/ResultsContainer/ResultsContainer.test.jsx]: It's the other way around - un-required props need default values.

File: src/Components/ResultsContainer/ResultsContainer.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/ResultsContainer/ResultsContainer.jsx]: I've noticed that too

File: src/sass/_results.scss (hunk 3 above)
Comment (mjoyce91) [reply src/sass/_results.scss]: We're just targeting IE11. I don't think that site is right - take a look at https://www.w3schools.com/cssref/sel_not.asp

Also seems to look fine when I tested it in IE11 anyways

**Pull request merged**